cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -66,6 +67,23 @@ var (
 		` + cmdutil.AlphaDisclaimer)
 )
 
+// certsData defines the behavior that a runtime data struct passed to the kubeconfig phase should
```

Comment:
Change from `passed to the kubeconfig phase`  to `passed to certs phase`

Reply:
Thanks!